src/app/charts/chart.component.ts

```diff
@@ -137,6 +158,11 @@ export class ChartComponent implements OnChanges {
         this.imageExportService.downloadAsPNG(this.chart.indicator.name, fileName);
     }
 
+    public onThresholdSelected($event) {
+        delete $event.event; // everything else is a threshold parameter
```

Comment:
Probably not a standard convention, just something I'd found I think from a blog post and one or two StackOverflow discussions, though I seem to have closed those tabs already. I'll try fiddling with it to come up with something better.